cypress/support/config/settings.js

```diff
@@ -1330,7 +1330,7 @@ module.exports = () => ({
       storyPage: {
         environments: {
           live: {
-            paths: ['/gahuza/51946134'],
+            paths: ['gahuza/amakuru-52821373'],
```

Comment:
```suggestion
            paths: ['/gahuza/amakuru-52821373'],
```

Reply:
Resolved now in my latest commit :)

Reply:
good spot 💯 
![image](https://user-images.githubusercontent.com/30599794/83280656-e4ab7c00-a1ce-11ea-9eec-7d2fa5ee16e5.png)